### ISSUE TITLE
Is this oriented, i can't make out

### ISSUE THREAD

**Issue by samirahmed** [Algorithmix/Papyrus#8]
These don't look oriented correctly, but I can' really tell, @crowell  can you verify this?
[full set here (testdrive/documents/oldthomas)](https://www.dropbox.com/sh/pm7640t4gd7mw4s/rqPqyuAuTx)

![image16](https://f.cloud.github.com/assets/839972/162777/dc4b8b22-77ce-11e2-9314-1a3c89087737.png)
![image17](https://f.cloud.github.com/assets/839972/162778/dc51cabe-77ce-11e2-9cda-a86ec1e33833.png)
![image18](https://f.cloud.github.com/assets/839972/162779/dc557c7c-77ce-11e2-904b-e24aa205f58f.png)

**Comment by samirahmed** [comment]
@crowell i think we can get orientation better than this... try the horizontal orientation thing in aforge and put it in as a parameter so that we can experiment please

**Comment by samirahmed** [comment]
@jmoisan can you also experiment with straightening (like rotating into upright position please checkout ) http://www.aforgenet.com/framework/docs/html/18d8bd4d-b15e-8507-a480-b2fe6eaeab44.htm

**Comment by jmoisan** [comment]
yes I'll see what I can do and check if that class works on our shreds

**Comment by crowell** [comment]
yeah, it still seems off, going to work on it tonight/tomorrow

On Tue, Apr 2, 2013 at 9:27 PM, Samir Ahmed notifications@github.comwrote:

> @crowell https://github.com/crowell i think we can get orientation
> better than this... try the horizontal orientation thing in aforge and put
> it in as a parameter so that we can experiment please
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Algorithmix/Papyrus/issues/8#issuecomment-15812879
> .

## 

Jeff Crowell
Boston University
Computer Engineering
https://github.com/crowell
